crates/runtime/src/env/mod.rs

```diff
@@ -50,7 +50,7 @@ pub trait EnvTypes {
     /// [`Template`] type.
     type TemplateStore: TemplateStore;
 
-    /// [`AccountStore`] type.
+    /// `AccountStore` type.
```

Comment:
Why are we removing the intra-doc links? Am I missing something?

Reply:
The `cargo doc` emitted warnings, probably I'm missing something

Reply:
Seems to be about `mod` visibility... I'm looking for a solution

Reply:
Ok, so the warning is actually a symptom, not a problem itself. We're exposing private `traits` within `EnvTypes`, which is public. `cargo-doc` doesn't like that because of the `rustdoc::private_intra_doc_links` lint. I should we should the public-private `EnvTypes` issue rather than removing intra-doc links.